src/ontology/mondo-edit.obo

```diff
@@ -500399,8 +500403,8 @@ xref: OMIM:619478 {source="MONDO:equivalentTo"}
 xref: UMLS:C5561960 {source="MEDGEN:1794170", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0001347 {source="DOID:0060918", source="OMIM:619478"} ! facioscapulohumeral muscular dystrophy
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2979 {source="OMIM:619478"} ! DNMT3B
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 {source="OMIM:619478"} ! DUX4
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2979 ! DNMT3B
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/50800 ! DUX4
```

Comment:
This disease contains "digenic" in the description and the gene associations should be removed (not just the source provenance).

Reply:
Per [explanation here](https://github.com/monarch-initiative/mondo/pull/8624#discussion_r1933035837), just because it has digenic in the label doesn't necessarily mean it will be removed. Though in most cases it will. However it'll be removed because there are >1 associations, not because 'digenic' is in the label.

Regarding removing the entire association, this is part of the `mondo` pipeline that Nico created. I assume you or I will edit this to remove the association when no evidence remains? I can do that if you want.

Reply:
MONDO:0030355 is in the [exclusion file](https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv) so these gene associations should not have existed in mondo-edit.obo. These gene associations should be removed.

Reply:
The gene association was removed.